code/ui/blocks/src/blocks/DocsPage.tsx

```diff
@@ -6,14 +6,22 @@ import { Description } from './Description';
 import { Primary } from './Primary';
 import { Controls } from './Controls';
 import { Stories } from './Stories';
+import { useOf } from './useOf';
 
-export const DocsPage: FC = () => (
-  <>
-    <Title />
-    <Subtitle />
-    <Description />
-    <Primary />
-    <Controls />
-    <Stories />
-  </>
-);
+export const DocsPage: FC = () => {
+  const resolvedOf = useOf('meta', ['meta']);
+  const { stories } = resolvedOf.csfFile;
```

Comment:
Maybe this should be `docsContext.componentStories()`, rather than digging into the `csfFile`?

Reply:
Yes I thought about that too, because that's how we do it elsewhere. But since we often share how the default DocsPage is implemented, I wanted to use the more official approach with `useOf`, instead of something that is semi-internal.